lib/subscriptions_test_kit/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SubscriptionsTestKit
-  VERSION = '0.9.0'.freeze
+  VERSION = '0.9.1'.freeze
```

Comment:
You have to run `bundle` after changing the version.

Reply:
done